src/SignalR/clients/ts/signalr/package.json

```diff
@@ -55,8 +55,5 @@
     "eventsource": "^1.0.7",
```

Comment:
This is the only mention of `eventsource` in a package.json file in this branch. The mention of `@types/eventsource` above is also unique. Should one or both be removed while I'm at it❔

Reply:
I think this is supposed to be here - should just update it to 1.1.1

Reply:
I'll just leave it alone because yarn.lock shows 1.1.2 anyhow.